src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -488,10 +494,22 @@ public void setTeamDomain(String teamDomain) {
             this.teamDomain = teamDomain;
         }
 
+        /**
+         * Deprecated for removal in 3.0
+         *
+         * Use tokenCredentialId instead
+         */
+        @Deprecated
         public String getToken() {
             return token;
         }
 
+        /**
+         * Deprecated for removal in 3.0
+         *
+         * Use tokenCredentialId instead
+         */
+        @Deprecated
         @DataBoundSetter
         public void setToken(String token) {
             this.token = token;
```

Comment:
Usually it is better to also enforce migration on the databound setter especially for JCasc :)

Reply:
Jcasc refuses to use deprecated properties unless you opt in.
But yes can add that here too